spikesorters/basesorter.py

```diff
@@ -138,51 +138,22 @@ def run(self, raise_error=True, joblib_backend=None):
                     self.parallel = False
                     print("RecordingExtractor objcts are not dumpable and can't be processed in parallel")
 
-        if raise_error:
+        try:
             if not self.parallel:
                 for i, recording in enumerate(self.recording_list):
                     self._run(recording, self.output_folders[i])
             else:
                 _ = Parallel(n_jobs=len(self.recording_list), backend=joblib_backend)(
```

Comment:
Do you dump this recording_list before ? I don't think the serilizer is done elswhere, no ?

Reply:
Look better (next line)

Reply:
oups sorry